Clinical-Genomics/patientMatcher
# Save Ensembl ids for demo patient genes automatically

### This PR adds | fixes:
- fix #211

### How to test:
- Check the difference when you load demo data with the command "pmatcher add demodata" in master and this branch

### Expected outcome:
- [ ] Demo genes won't have Ensembl IDs when loaded with master branch. They will have Ensembl IDs when loaded using this branch

### Review:

This [version](https://semver.org/) is a:
